WeakAuras/GenericTrigger.lua

```diff
@@ -1005,6 +1103,22 @@ function GenericTrigger.Add(data, region)
                 -- that are already in place. Replacing all those checks would be a pain in the ass.
                 trigger_events[index] = "COMBAT_LOG_EVENT_UNFILTERED_CUSTOM";
               end
+              -- custom events in the form of event:unit1:unit2:unitX are registered with RegisterUnitEvent
+              local trueEvent
+              for i in event:gmatch("[^:]+") do
+                 if not trueEvent then
+                    trueEvent = string.upper(i)
+                 else
+                    local unit = string.lower(i)
+                    if WeakAuras.baseUnitId[unit] then
+                      trigger_unit_events[unit] = trigger_unit_events[unit] or {}
+                      tinsert(trigger_unit_events[unit], trueEvent)
+                      trigger_events[index] = nil
+                    else
+                      print(L["Unit %s is not a valid unit for RegisterUnitEvent"]:format(unit))
```

Comment:
i guess we want to show a red message under the box instead of printing this in chat ?

Reply:
done